addons/godot-playfab/PlayFabClientConfig/PlayFabClientConfigLoader.gd

```diff
@@ -0,0 +1,55 @@
+extends Reference
+class_name PlayFabClientConfigLoader
+
+
+const DEBUG_DO_NOT_ENCRYPT = false	# Only works on debug builds
+const SECTION_NAME = "PlayFab"
+
+
+var _load_path = "user://playfab_client_config.cfg"
+var _config: ConfigFile = ConfigFile.new()
+var _title_id: String
+var _errors = []
+
+
+func save(title_id: String, new_config: PlayFabClientConfig):
+	_config.set_value(SECTION_NAME, "session_ticket", new_config.session_ticket)
+	_config.set_value(SECTION_NAME, "master_player_account_id", new_config.master_player_account_id)
+	_config.set_value(SECTION_NAME, "entity_token", new_config.entity_token)
+	_config.set_value(SECTION_NAME, "login_id", new_config.login_id)
+	_config.set_value(SECTION_NAME, "login_type", new_config.login_type)
+	_config.set_value(SECTION_NAME, "login_timestamp", new_config.login_timestamp)
+
+	if OS.is_debug_build() && DEBUG_DO_NOT_ENCRYPT:
+		_config.save(_load_path)
+	else:
+		var password = _title_id # TODO: Rename _title_id to password
+		_config.save_encrypted_pass(_load_path, password)
+
+
+func load(title_id: String) -> PlayFabClientConfig:
+	_config = ConfigFile.new()
+	var new_config = PlayFabClientConfig.new()
+	var err: int
+
+	if OS.is_debug_build() && DEBUG_DO_NOT_ENCRYPT:
+		err = _config.load(_load_path)
+	else:
+		var password = _title_id # TODO: Rename _title_id to password
+		err = _config.load_encrypted_pass(_load_path, password)
+
+	# If the file didn't load, ignore it.
+	if err != OK:
+		var error_message = "Config file didn't load. Error code: %f" % err
+		print_debug(error_message)
+		_errors.append(error_message)
+		return PlayFabClientConfig.new()
+
+	new_config.session_ticket = _config.get_value(SECTION_NAME, "session_ticket")
```

Comment:
Can we simplify the get_value calls to one generic call and iterate over members?